templates/zerver/emails/followup_day1.txt

```diff
@@ -1,33 +1,43 @@
-Hi there,
+{{ _('Hi there,') }}
 
-Welcome to Zulip! A few tips to get you started:
+{{ _('Welcome to Zulip! A few tips to get you started:') }}
 
+{% trans %}
 * Zulip works best when it's always open, so we suggest pinning a browser
-tab or downloading our desktop and mobile apps (https://zulipchat.com/apps).
+  tab or downloading our desktop and mobile apps (https://zulipchat.com/apps).
+{% endtrans %}
 
+{% trans %}
 You'll need an Organization URL when installing the apps. That's
 {{ realm_uri }}
+{% endtrans %}
 
-* Become a Zulip pro with a few keyboard shortcuts:
-  n: Next unread thread
-  r: Reply to message under the blue box
-  c: Start a new topic
+* {{ _('Become a Zulip pro with a few keyboard shortcuts:') }}
+  n: {{ _('Next unread thread') }}
+  r: {{ _('Reply to message under the blue box') }}
+  c: {{ _('Start a new topic') }}
 
+{% trans %}
 * Check out our guide for new users:
 {% if is_realm_admin %}
 {{ organization_setup_advice_link }}.
 {% else %}
 {{ getting_started_with_zulip_link }}.
 {% endif %}
+{% endtrans %}
```

Comment:
You can't put Django template logic inside a translation tag.  I think what we need to do for this is move the picking-which-link logic into the Python code, and then just have this be `getting_started_link` (and always be the correct one either way).